external/cmapscompress/compress.js

```diff
@@ -204,6 +204,7 @@ function parseCMap(binaryData) {
         bufferSize = 0;
       while (s.length < lengthInChars) {
         while (bufferSize < 4 && stack.length > 0) {
+          // eslint-disable-next-line operator-assignment
```

Comment:
Why do we need these ignore lines in this commit? The bitwise OR and XOR operations used here should be commutative and therefore the order doesn't matter, so I think we can also use `buffer |= (stack.pop() << bufferSize)` here?

Reply:
> Why do we need these ignore lines in this commit? The bitwise OR and XOR operations used here should be commutative and therefore the order doesn't matter,

Because I was being dumb and didn't realize that those are in fact commutative :-)